python_modules/dagster/dagster/_core/execution/asset_backfill.py

```diff
@@ -627,7 +628,15 @@ def execute_asset_backfill_iteration(
             if unloadable_locations
             else ""
         )
-        raise DagsterAssetBackfillDataLoadError(f"{ex}. {unloadable_locations_error}")
+        if os.environ.get("DAGSTER_BACKFILL_RETRY_DEFINITION_CHANGED_ERROR"):
+            logger.error(
+                "Retrying error in backfill"
+                f" {backfill.backfill_id}:\n{ex}. {unloadable_locations_error}"
```

Comment:
Language tweak suggestion to mimic the corresponding feature in the scheduler that retries once user code is available again:


"Backfill was unable to continue due to a missing asset or partition in the asset graph. The backfill will resume once it is available again. {unloadable_locations_error}"